_posts/2025-11-11-chip-on-board-strategy.md

```
@@ -0,0 +1,264 @@
+
+
+Panelized for efficient handling and assembly by wirebonding partners:
+
+{% include post-gallery.html gallery="mezzanine_panels" %}
+
+
+The standard configuration can be found in the [GitHub repository](https://github.com/wafer-space/gf180mcu-project-template/blob/main/librelane/config.yaml). For easier viewing, it's also available as a [Google Sheet](https://docs.google.com/spreadsheets/d/1pI2BAEWEexXcXN3vah3SR85zPIV6eAXPGXc2bcvoSGU/edit?gid=0#gid=0) that shows the complete pad mapping.
+
+![Die padring layout with 74-pad configuration](/assets/images/news/chip-on-board-strategy/padframes-cob-die-padring.png)
+
+To help visualize your own pad layouts, Discord user **@Tholin** has developed a Python-based pad viewer that generates an image of the pad layout defined in your template. This is useful for verifying your configuration matches your design intent. You can find the tool [here](https://github.com/AvalonSemiconductors/ws-submission-2025/blob/main/padout_gen.py).
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)